services/resetPassword.js

```diff
@@ -0,0 +1,24 @@
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+const userRepository = require('../repositories/user');
+const errors = require('../configuration/errors');
+const env = require("../configuration/env")
+
+async function resetPassword(token, newPassword, confirmPassword) {
```

Comment:
we don't need to take confirm password because we will validate that on frontend.